eclipse-collections/src/main/java/org/eclipse/collections/impl/block/procedure/NonMutatingAggregationProcedure.java

```diff
@@ -47,4 +47,12 @@ public void value(T each)
         K key = this.groupBy.valueOf(each);
         this.map.updateValueWith(key, this.zeroValueFactory, this.nonMutatingAggregator, each);
     }
+
+    public void forEachWithOccurrences(int occurrences, T each)
```

Comment:
Hi @prathasirisha, if you switch the parameters you can use a method reference to call this.

Reply:
Yup, done!